### ISSUE TITLE
USA's new guidelines to enhance the integrity of voluntary carbon markets (VCMs). Would this affect SCER?

### ISSUE THREAD

**Issue by chrisxie-fw** [Green-Software-Foundation/scer#68]
The Biden administration has introduced new guidelines to enhance the integrity of voluntary carbon markets (VCMs). These guidelines ensure carbon credits represent real, additional, and lasting emissions reductions. Key elements include prioritizing emissions reductions within corporate operations, ensuring transparency through public disclosures, and setting high standards for the creation and certification of carbon credits. The initiative aims to restore trust in VCMs and support private investment in climate action, despite ongoing criticisms regarding the effectiveness and transparency of existing practices.

Would this affect SCER?

Sources:

1. [Can carbon offsets actually work? The Biden administration thinks so. | Grist](https://grist.org/regulation/carbon-offsets-credits-voluntary-carbon-market-biden-administration/) 
2. [Biden Administration Endorses Voluntary Carbon Offsets | ESG Legal Solutions, LLC](https://www.esglegalsolutions.com/2024/06/biden-administration-endorses-voluntary-carbon-offsets/) 
3. [Biden Administration Issues New Guidelines In Buying Carbon Offsets To Ensure Integrity | IBTimes](https://www.ibtimes.com/biden-administration-issues-new-guidelines-buying-carbon-offsets-ensure-integrity-3733035).

**Comment by jawache** [comment]
@chrisxie-fw offsets is a subject with a deep lack of consensus, we've had discussions around the use of market based instruments like these during the development of the SCI. I've tried to express them here #66. 

I would say if SCER does become a meta-standard for labeling different types of environmental impact reporting standards it should just adhere to whatever the underlying standard is. I.e. if it's labeling for the SCI protocol then numbers needs to be calculated using the SCI which doesn't allow offsets. If it's a label say for a GHG value, then it should adhere to the GHG protocol (which allows energy offsets only for scope 2) and you have to label your footprint with market-based to indicate you are including offsets etc...

Related to that I think that's a good way for SCER to fit into the ecosystem of standards. I don't believe SCER can both define a new standard for measuring software AND be a labeling system for existing standards. It could just be a link to an existing standard (SCI, GHG, E*AI etc...) and be a signal (label) that someone is following the rules of the underling standard.

**Comment by chrisxie-fw** [comment]
Agreed! At this stage, SCER is not dictating a specific approach but rather respecting the terms of the underlying standard. We should ensure that the Standard is not overly restrictive, making it easier for people to adopt and follow. A standard adopted by the industry holds much more value and impact than a comprehensive but restrictive one that is difficult to adhere to. This design principle guides me in developing the SCER standard: to make it easier for organizations to adopt. The [SCER Certification Program](SCER_Certification_Program/SCER_Certification_Prorgram_for_LLMs.md) further incentivizes adoption by offering a label with tangible business value. This is why we don’t just create standards but also develop programs that reward organizations for adopting them. I hope this makes sense and resonates with the community.

**Comment by jawache** [comment]
> At this stage, SCER is not dictating a specific approach but rather respecting the terms of the underlying standard.

In that case @chrisxie-fw I'm quite confused, because in the spec as it's written right now you are proposing several new standards for measurement. For Databases it's [Ops/Watt-hour](https://github.com/Green-Software-Foundation/scer/blob/Dev/Software_Carbon_Efficiency_Rating(SCER)/SCER%20for%20Database%20Server%20Software.md#6-benchmarking) and for Music Streaming it's [Streams/Watt-hour](https://github.com/Green-Software-Foundation/scer/blob/Dev/Software_Carbon_Efficiency_Rating(SCER)/SCER%20for%20Music%20Streaming%20Services.md#6-benchmarking) but there are no standards for Ops/Watt-hour or Streams/Watt-hour that I am aware of? 

That's what I was referring to when I said 
>  I don't believe SCER can both define a new standard for measuring software AND be a labeling system for existing standards 

To conform to the SCI the way you would represent both of those _might_ be _Carbon per OPS_ for Databases and _Carbon per Stream_ for Music Streaming. If they were shifted to that type of calculation then it would align to the SCI specification. SCI is very specifically "Carbon per XXX".

**Comment by chrisxie-fw** [comment]
Thank you @jawache for your comments! 

The two specs (database and music streaming services) are meant to be **example** use cases of the base SCER spec to create category specific SCER spec, they are not actual spec's for databases and streaming services. This was mentioned in the [base spec](https://github.com/Green-Software-Foundation/scer/blob/Dev/Software_Carbon_Efficiency_Rating(SCER)/Software_Carbon_Efficiency_Rating_Specification.md) section 3: ....A separate document describes an example of how to use the base specification to create a category specific specification, and examples were given on how to create the category specific specification.

I agree that the titles of the document should be more explicit to avoid confusions.

I also agree with you that if we indeed create an actual SCER spec for databases, carbon/ops will be prefered, because carbon efficiencies are assessed here.

**Comment by jawache** [comment]
Thanks @chrisxie-fw, yes I think some more clear naming would help, if I'm confused others will be confused also 😀

In addition it should also just be possible to rewrite those examples in the SCI format. E.g. Ops/Watt-hour can be rewriten as Carbon/OP but we simply use fixed coefficients for I and M (the spec allows for that). That way your examples still work functionally are they area, minimal changes, and since everyone here all are very familiar with the SCI it would go a long way to helping people feel more familiar with SCER. I'm happy to give an attempt at the update if you want, i can submit a PR for review next meeting?

**Comment by chrisxie-fw** [comment]
@jawache thank you so much! It would be perfect to have you update the doc as an SCI founder and expert!
